ext/afform/admin/ang/afGuiEditor/afGuiFieldValue.directive.js

```diff
@@ -38,7 +38,7 @@
               // Static options for choosing current user or other entities on the form
               options = [];
               filters = (field.input_attrs && field.input_attrs.filter) || {};
-              if (field.fk_entity === 'Contact' && (!filters.contact_type || filters.contact_type === 'Individual')) {
+              if (field.fk_entity === 'Individual' || (field.fk_entity === 'Contact' && (!filters.contact_type || filters.contact_type === 'Individual'))) {
                 options.push('user_contact_id');
               }
               _.each(ctrl.editor ? ctrl.editor.getEntities({type: field.fk_entity}) : [], function(entity) {
```

Comment:
I think this is the last bug -- for fields with fk_entity === 'Contact' the type: 'Contact' isn't catching afform entities with type: Individual / Household / Organization 

maybe [afGuiEditor.getEntities](https://github.com/colemanw/civicrm-core/blob/14444414795303e10633c48732782b8821423f5f/ext/afform/admin/ang/afGuiEditor/afGuiEditor.component.js#L331) could catch type: 'Contact' and translate to union of type: 'Individual' + type: 'Household' +... ?